blackjack.js

```diff
@@ -30,7 +73,13 @@ const calcPoints = (hand) => {
  * @returns {boolean} whether dealer should draw another card
  */
 const dealerShouldDraw = (dealerHand) => {
-  // CREATE FUNCTION HERE
+  let dealerTotal = function dealerTotalFunc (dealerHand){
+    calcPoints();
+    return (score > 16);
+  }
+
+  return dealerTotal;
```

Comment:
since `dealerTotal` is a function that is declared inside the `dealerShouldDraw` function, what we end up with here is that `dealerShouldDraw` actually returns the function . . . but since it is actally a value then when it is evaluated on line 147, it _always_ looks to be `true` . . . that's why the program does not break.

JavaScript is really flexible, so it will totally allow weird things like this to go through and the result is a sort of false-positive and no errors getting thrown. 

This function should get a full refactor, but here's the steps you are aiming for:

It should take a 'hand' object as an argument . . . .
Then store a const that is the object returned when you call `calcPoints(hand)` . . . .
Then a bit of if/else logic comparing the total to the dealer rules, to result in returning a boolean.